jackal_hulk:
      diff_drive_controller:
         type: "diff_drive_controller/DiffDriveController"
         publish_rate: 100

         
         right_wheel: ['joint_4', 'joint_5']
         left_wheel: ['joint_2','joint_3']

         wheel_radius: 0.1
         wheel_separation: 0.32378
         pose_covariance_diagonal: [0.001, 0.001, 1000.0, 1000.0, 1000.0, 1000.0]
         twist_covariance_diagonal: [0.001, 0.001, 1000.0, 1000.0, 1000.0, 1000.0]
         base_frame_id: base_link

         linear:
          x:
            has_velocity_limits    : true
            max_velocity           : 2 # m/s
            min_velocity           : -2 # m/s
            has_acceleration_limits: true
            max_acceleration       : 1 # m/s^2
            min_acceleration       : -1 # m/s^2
         


         angular:
          z:
            has_velocity_limits    : true
            max_velocity           : 1  # rad/s
            has_acceleration_limits: true
            max_acceleration       : 1  # rad/s^2
            has_jerk_limits        : true
            max_jerk               : 1  # rad/s^3

      hulk_controller:
            # type: position_controllers/JointTrajectoryController
            # joints:
            #       - joint_6
            #       - joint_7
            #       - joint_8
            #       - joint_9
            #       - joint_10
            #       - joint_11
 
            # stop_trajectory_duration: 5
            # state_publish_rate:  30

            joint6_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_6
                  pid: {p: 1.0, i: 0.1, d: 0.1}   

            joint7_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_7
                  pid: {p: 1.0, i: 0.1, d: 0.1}         

            joint8_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_8
                  pid: {p: 1.0, i: 0.1, d: 0.1}         

            joint9_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_9
                  pid: {p: 1.0, i: 0.1, d: 0.1}         

            joint10_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_10
                  pid: {p: 1.0, i: 0.1, d: 0.1}         

            joint11_position_controller:
                  type: "position_controllers/JointPositionController"
                  joint: joint_11
                  pid: {p: 1.0, i: 0.1, d: 0.1}    

  
